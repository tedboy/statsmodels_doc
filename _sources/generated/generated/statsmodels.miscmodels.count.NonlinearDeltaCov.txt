statsmodels.miscmodels.count.NonlinearDeltaCov
==============================================

.. currentmodule:: statsmodels.miscmodels.count
.. autoclass:: NonlinearDeltaCov
   :members: __init__
   :undoc-members:


Methods
-------
.. autosummary::
   :toctree:


   ~NonlinearDeltaCov.__init__
   ~NonlinearDeltaCov.cov
   ~NonlinearDeltaCov.expected
   ~NonlinearDeltaCov.grad
   ~NonlinearDeltaCov.wald





