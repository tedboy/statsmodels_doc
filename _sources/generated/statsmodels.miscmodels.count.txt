statsmodels.miscmodels.count
============================

.. automodule:: statsmodels.miscmodels.count

   
   
   Functions
   ---------
   .. autosummary::
      :toctree:

   
      maxabs
      maxabsrel
   
   

   
   
   Classes
   ---------
   .. autosummary::
      :toctree:generated/
      :template:class_custom.rst

   
      GenericLikelihoodModel
      NonlinearDeltaCov
      PoissonGMLE
      PoissonOffsetGMLE
      PoissonZiGMLE

   .. toctree::
       :maxdepth: 1
       :hidden:

   
       generated/statsmodels.miscmodels.count.GenericLikelihoodModel
       generated/statsmodels.miscmodels.count.NonlinearDeltaCov
       generated/statsmodels.miscmodels.count.PoissonGMLE
       generated/statsmodels.miscmodels.count.PoissonOffsetGMLE
       generated/statsmodels.miscmodels.count.PoissonZiGMLE

   
   

   
   
   