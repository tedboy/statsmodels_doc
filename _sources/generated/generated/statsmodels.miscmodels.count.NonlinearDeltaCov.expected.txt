statsmodels.miscmodels.count.NonlinearDeltaCov.expected
=======================================================

.. currentmodule:: statsmodels.miscmodels.count

.. automethod:: NonlinearDeltaCov.expected